client/payment-details/payment-method/becs/index.js

```diff
@@ -0,0 +1,134 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import PaymentDetailsPaymentMethodDetail from '../detail';
+
+/**
+ * Extracts and formats payment method details from a charge.
+ *
+ * @param {Object} charge The charge object.
+ * @return {Object}       A flat hash of all necessary values.
+ */
+const formatPaymentMethodDetails = ( charge ) => {
+	const { billing_details: billingDetails, payment_method: id } = charge;
+	const {
+		bsb_number: bsb,
+		last4,
+		fingerprint,
+	} = charge.payment_method_details.au_becs_debit;
+
+	const { name, email, formatted_address: formattedAddress } = billingDetails;
+
+	return {
+		bsb,
+		last4,
+		fingerprint,
+		id,
+		name,
+		email,
+		formattedAddress,
+	};
+};
+
+/**
+ * Placeholders to display while loading.
+ */
+const paymentMethodPlaceholders = {
+	bsb: '000000',
+	last4: '0000',
+	fingerprint: 'fingerprint placeholder',
+	id: 'id placeholder',
+	name: 'name placeholder',
+	email: 'email placeholder',
+	formattedAddress: 'address placeholder',
+};
+
+const SepaDetails = ( { charge = {}, isLoading } ) => {
```

Comment:
SepaDetails is mentioned here and at the bottom of the file.

Reply:
Thanks! I missed it, addressed in 824c9e3